docs/topics/vs-code-extension-interface.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id="vs-code-extension-interface_{context}"]
+= {ProductShortName} extension interface
+
+The interface of the {ProductName} ({ProductShortName}) extension includes the following components:
+
+* The left pane displays the applications and files analyzed by MTA and any issues uncovered by the analysis. The pane includes a folder icon named *Report*, which you can click to open the MTA Report in your browser.
```

Comment:
I think "uncovered" sounds strange in this context. Try to find a less ambiguous term.

"folder icon named Report" - verbose. Report folder icon?

Also, the procedures have a lot of anthromorphic expressions. Panes do not really "do" anything. Try to rewrite so that the user (You can view the applications and files... You can click the ...) is doing the action.